tasks/configuration.yml

```diff
@@ -3,10 +3,27 @@
 
 - name: list of required SAP Notes
   debug:
-    var: sap_netweaver_preconfigure_sapnotes | difference([''])
+    var: __sap_netweaver_preconfigure_sapnotes | difference([''])
 
 - name: include configuration actions for required sapnotes
   include_tasks: "sapnote/{{ item }}.yml"
-  with_items: "{{ sap_netweaver_preconfigure_sapnotes | difference(['']) }}"
+  with_items: "{{ __sap_netweaver_preconfigure_sapnotes | difference(['']) }}"
+
+- name: Warn if not enough swap space is configured
+  fail:
+    msg: "The system has only {{ ansible_swaptotal_mb }} MB of swap space configured,
+    which is less than the minimum required amount of 20480 MB for SAP NetWeaver!"
+  ignore_errors: yes
+  when:
+    - ansible_swaptotal_mb < 20480
+    - not sap_netweaver_preconfigure_fail_if_not_enough_swap_space_configured|d(true)
```

Comment:
should this be `sap_netweaver_preconfigure_warn_if_not_enough_swap_space_configured` ?

Reply:
No, as I want the role to either fail (in case sap_netweaver_preconfigure_fail_if_not_enough_swap_space_configured is undefined to "yes") or warn (in case sap_netweaver_preconfigure_fail_if_not_enough_swap_space_configured is "no").